### PR TITLE
Add shop.inc in jwp_skin

### PR DESCRIPTION
**Pull request by WhyFaust**

### CODE REVIEW

**Review by TiBarification — CHANGES_REQUESTED** [review]
Remove this file (shop.inc), cause it is old version of it.
Use reference to fork repository by R1KO.
I'm thinking about reconstruction builder (travis-ci),

**Comment by TiBarification** [comment]
@mrkos9i4ok you need to add support if server does not have [Shop] Core plugin. I added support for Fork by @R1KO.

File: .travis.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ before_deploy:
     - mv addons/sourcemod/scripting/include/csgo_colors.inc jwp/addons/sourcemod/scripting/include/
     - mv addons/sourcemod/scripting/include/emitsoundany.inc jwp/addons/sourcemod/scripting/include/
     - mv addons/sourcemod/scripting/include/multicolors* jwp/addons/sourcemod/scripting/include/
-    - mv addons/sourcemod/scripting/include/n_arm_fix.inc jwp/addons/sourcemod/scripting/include/
+    - mv addons/sourcemod/scripting/include/n_arms_fix.inc jwp/addons/sourcemod/scripting/include/
```
Comment (TiBarification) [inline .travis.yml]: Your code is not up to date. Please use latest commit for your changes.

**Review by TiBarification — DISMISSED** [review]

**Comment by WhyFaust** [comment]
What `not travis`

**Comment by TiBarification** [comment]
I help you to fix this commit.

**Comment by TiBarification** [comment]
@mrkos9i4ok fixed integration with jwp_skin module.

**Comment by WhyFaust** [comment]
Still have to know how)))

**Comment by TiBarification** [comment]
Build is failed, because your module jwp_ratio uses lib `mystocks` that isn't default and not in list of travis. Add this lib manually, or just add a simple functions that it uses.

**Comment by TiBarification** [comment]
Congradulations with passed build.

**Comment by WhyFaust** [comment]
Now can this be closed?
And how to edit the files so that they do not show up here?

**Pull request merged**